packages/neuron-wallet/tests/services/address.test.ts

```diff
@@ -310,7 +310,8 @@ describe('integration tests for AddressService', () => {
 
     describe('#generateAndSaveForPublicKey', () => {
       describe('with public key info exist for the public key', () => {
-        const publicKey = 'public key'
+        // const publicKey = 'public key' // not valid public key
+        const publicKey = '0x' + '00'.repeat(65) // 65 bytes pub key
```

Comment:
It can't pass test after refactor, because `publicKey` is used to generate secp256k1 `args`, in `lumos` we can call `utils.ckbHash(pubkey).slice(0, 42)`, which requires `publicKey` to be a valid `BytesLike` value.